config.yaml

```diff
@@ -5,26 +5,37 @@ results_dir: results/
 summary_dir: results/
 costs_dir: data/ #TODO change to the equivalent of technology data
 
-run: Runname_test
+run: test_again_10
 
 foresight: overnight
 
 scenario:
   simpl: # only relevant for PyPSA-Eur
   - ""
   clusters: # number of nodes in Europe, any integer between 37 (1 node per country-zone) and several hundred
-  - 20
+  - 1010
   planning_horizons: # investment years for myopic and perfect; or costs year for overnight
   - 2030
   ll:
   - "c1.0"
   opts:
   - "Co2L"
   sopts:
-  - "730H"
+  - "3H"
+  
 
 countries: ['MA']
 
+custom_data:
```

Comment:
Please add all changes in this file also to `https://github.com/pypsa-meets-africa/pypsa-earth-sec/blob/main/test/config.test1.yaml`

Reply:
Otherwise the CI fails